internal/terraform/exec/exec_test.go

```diff
@@ -70,7 +70,7 @@ func newExecutor(t *testing.T) TerraformExecutor {
 		t.Fatal(err)
 	}
 
-	opts := tfinstall.ExactVersion("0.13.1", installDir)
+	opts := tfinstall.ExactVersion("1.1.0", installDir)
```

Comment:
For posterity `1.0.2` was the first version to support `darwin/arm64`:
https://releases.hashicorp.com/terraform/1.0.2/